src/pythermalcomfort/zhang_comfort_model_functions/coefficients.py

```diff
@@ -0,0 +1,479 @@
+import pandas as pd  # for making dataframe
+
+c1_plus = {
+    "head": 0.9,
```

Comment:
These should be data classes and not dictionaries since we have the same string repeated many times. You can create a dataclass and then use it to define `c1_plus`, `c1-minus`, ....

Moreover the variable name c1_plus is not very descriptive and we could improve it

Reply:
Agree that we should use dataclass, I will do it later.